README.md

```diff
@@ -268,6 +268,35 @@ keep it.
 Use [proofit404/blacken](https://github.com/proofit404/blacken).
 
 
+### PyCharm
+
+(Adapted from [*Pylinting with PyCharm*](https://kirankoduru.github.io/python/pylint-with-pycharm.html))
```

Comment:
I understand why you want to share credit but I think the steps outlined below are generic enough that you don't have to. This just adds noise to the guide.

Reply:
I agree with @ambv on this point